### PR TITLE
✨ feat: migrate less to token for Slider

### PR DESCRIPTION
**Pull request by heiyu4585**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      migrate less to token for Slider     |
| 🇨🇳 Chinese |       修改 组件 less token     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 85e13ab</samp>

This pull request adds component token support for the slider component, which enables users to customize its style through the `ConfigProvider` component. It also adds a component token demo and documentation for the slider component in both Chinese and English languages. It updates the migrate less variables documentation to include the slider component.

### <samp>🤖 Generated by Copilot at 85e13ab</samp>

*  Add a component token demo for the slider component that shows how to customize the style of the slider using the ConfigProvider component and the component token interface ([link](https://github.com/ant-design/ant-design/pull/42428/files?diff=unified&w=0#diff-ed4affc4a607a6fe16e47e8755bf69c6b627052e448a587fe9bd83efa1058af1R1-R7), [link](https://github.com/ant-design/ant-design/pull/42428/files?diff=unified&w=0#diff-2f284986ec3fa07ff2f46217cd1872f47acee7381b8b10348930be98c8282b79R1-R36), [link](https://github.com/ant-design/ant-design/pull/42428/files?diff=unified&w=0#diff-1923e5c169ac9e00e9733ef60f83f56bdbd01690c370fa8071a0fd2d0e4581c3R30), [link](https://github.com/ant-design/ant-design/pull/42428/files?diff=unified&w=0#diff-6e1f3b862cb377975f6bca16d005791420d2467cb8d47d9bcce69f3bcbf91cb7R31), [link](https://github.com/ant-design/ant-design/pull/42428/files?diff=unified&w=0#diff-9b9560a611e7ed0e6ef24ca9f1faff1e8c816d3f35ed6a1f73c36d2b42790abaR22-R40), [link](https://github.com/ant-design/ant-design/pull/42428/files?diff=unified&w=0#diff-9b9560a611e7ed0e6ef24ca9f1faff1e8c816d3f35ed6a1f73c36d2b42790abaR361-R379))
*  Update the slider component documentation in both languages to include the component token demo in the examples section, using the debug attribute to enable the component token debug mode ([link](https://github.com/ant-design/ant-design/pull/42428/files?diff=unified&w=0#diff-1923e5c169ac9e00e9733ef60f83f56bdbd01690c370fa8071a0fd2d0e4581c3R30), [link](https://github.com/ant-design/ant-design/pull/42428/files?diff=unified&w=0#diff-6e1f3b862cb377975f6bca16d005791420d2467cb8d47d9bcce69f3bcbf91cb7R31))
*  Update the migrate less variables documentation in both languages to include the slider component in the result section, using a table to show the mapping between the Less variables and the component token properties, and the remarks for each property ([link](https://github.com/ant-design/ant-design/pull/42428/files?diff=unified&w=0#diff-999566baa64628c49813370e7cb13c3c8f5e0c99ef8fe9cec7c5052985c10d9aL140-R165), [link](https://github.com/ant-design/ant-design/pull/42428/files?diff=unified&w=0#diff-dff3e0a0a5fae848621e55da4e2adaf0ec6c57d5e29d3f601799c8898c280739L140-R165))

### CODE REVIEW

File: docs/react/migrate-less-variables.zh-CN.md [hunk 1]
```diff
@@ -256,7 +256,31 @@ title: Less 变量迁移 Design Token
 | `@skeleton-paragraph-li-height` | `paragraphLiHeight` | - |
 | `@skeleton-paragraph-li-margin-top` | - | 由于样式变化已废弃 |
 
-<!-- ### Slider 滑动输入条 -->
+### Slider 滑动输入条
+
+<!-- prettier-ignore -->
+| Less 变量 | Component Token | 备注 |
+| --- | --- | --- |
+| `@slider-margin` | `iconFontSize` | - |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.zh-CN.md:264]:
```suggestion
| `@slider-margin` | - | 可由 `className` 或 `style` 直接修改 |
```

File: docs/react/migrate-less-variables.zh-CN.md [hunk 2]
```diff
@@ -256,7 +256,31 @@ title: Less 变量迁移 Design Token
...
+| `@slider-handle-color-hover` | `handleColorHover` | - |
+| `@slider-handle-color-focus` | `handleColorFocus` | - |
+| `@slider-handle-color-focus-shadow` | `handleColorFocusShadow` | - |
+| `@slider-handle-color-tooltip-open` | `handleColorTooltipOpen` | - |
+| `@slider-handle-size` | `handleSize` | - |
+| `@slider-handle-margin-top` | `handleMarginTop` | - |
+| `@slider-handle-margin-left` | `handleMarginLeft` | - |
+| `@slider-handle-shadow` | `handleShadow` | - |
+| `@slider-dot-border-color` | `dotBorderColor` | - |
+| `@slider-dot-border-color-active` | `dotBorderColorActive` | - |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.zh-CN.md:281]:
```suggestion
| `@slider-rail-background-color` | `railBg` | - |
| `@slider-rail-background-color-hover` | `railHoverBg` | - |
| `@slider-track-background-color` | `trackBg` | - |
| `@slider-track-background-color-hover` | `trackHoverBg` | - |
| `@slider-handle-border-width` | `handleLineWidth` | - |
| `@slider-handle-background-color` | `handleBg` | - |
| `@slider-handle-color` | `handleColor` | - |
| `@slider-handle-color-hover` | `handleHoverColor` | - |
| `@slider-handle-color-focus` | `handleActiveColor` | - |
| `@slider-handle-color-focus-shadow` | - | 样式变化已废弃 |
| `@slider-handle-color-tooltip-open` | `handleActiveColor` | - |
| `@slider-handle-size` | `handleSize` | - |
| `@slider-handle-margin-top` | `handleMarginTop` | - |
| `@slider-handle-margin-left` | `handleMarginLeft` | - |
| `@slider-handle-shadow` | - | 样式变化已废弃 |
| `@slider-dot-border-color` | `dotBorderColor` | - |
| `@slider-dot-border-color-active` | `dotActiveBorderColor` | - |
```

File: docs/react/migrate-less-variables.zh-CN.md [hunk 3]
```diff
@@ -256,7 +256,31 @@ title: Less 变量迁移 Design Token
...
+| `@slider-disabled-color` | `disabledColor` | - |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.zh-CN.md:282]:
```suggestion
| `@slider-disabled-color` | `trackBgDisabled` | - |
```

File: docs/react/migrate-less-variables.zh-CN.md [hunk 4]
```diff
@@ -256,7 +256,31 @@ title: Less 变量迁移 Design Token
...
+| `@slider-disabled-background-color` | `disabledBackgroundColor` | - |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.zh-CN.md:283]: 这个 v4 也没有用过了，可以废弃

File: docs/react/migrate-less-variables.zh-CN.md (hunk 2 above)
Comment (heiyu4585) [reply docs/react/migrate-less-variables.zh-CN.md:281]: handleBg 没有使用了

File: docs/react/migrate-less-variables.zh-CN.md (hunk 3 above)
Comment (heiyu4585) [reply docs/react/migrate-less-variables.zh-CN.md]: done

File: components/slider/style/index.tsx [hunk 5]
```diff
@@ -19,6 +19,18 @@ export interface ComponentToken {
   handleLineWidth: number;
   handleLineWidthHover: number;
   dotSize: number;
+  railBg: string;
+  railHoverBg: string;
+  trackBg: string;
+  trackHoverBg: string;
+  handleColor: string;
+  handleHoverColor: string;
+  handleColorTooltipOpen: string;
```
Comment (MadCcc) [inline components/slider/style/index.tsx:28]: 这个应该改成 handleActiveColor 了

File: components/slider/style/index.tsx [hunk 6]
```diff
@@ -339,6 +349,16 @@ export default genComponentStyleHook(
...
       handleLineWidth,
       handleLineWidthHover,
+      railBg: token.colorFillTertiary,
+      railHoverBg: token.colorFillSecondary,
+      trackBg: token.colorPrimaryBorder,
+      trackHoverBg: token.colorPrimaryBorderHover,
+      handleColor: token.colorPrimaryBorder,
+      handleHoverColor: token.colorPrimaryHover,
+      handleActiveColor: token.colorPrimary,
+      dotBorderColor: '#303030',
```
Comment (MadCcc) [inline components/slider/style/index.tsx:359]: 这个没用过

File: components/slider/style/index.tsx [hunk 7]
```diff
@@ -177,7 +187,7 @@ const genBaseStyle: GenerateStyle<SliderToken> = (token) => {
         position: 'absolute',
         width: dotSize,
         height: dotSize,
-        backgroundColor: token.colorBgElevated,
+        backgroundColor: token.dotBorderColor,
```
Comment (MadCcc) [inline components/slider/style/index.tsx:190]: 用错地方了，是 borderColor 才对

File: components/slider/style/index.tsx [hunk 8]
```diff
@@ -193,24 +203,24 @@ const genBaseStyle: GenerateStyle<SliderToken> = (token) => {
...
           backgroundColor: token.colorBgElevated,
-          borderColor: token.colorTextDisabled,
+          borderColor: token.trackBgDisabled,
           boxShadow: 'none',
           cursor: 'not-allowed',
         },
 
         [`${componentCls}-handle::after`]: {
-          backgroundColor: token.colorBgElevated,
+          backgroundColor: token.handleActiveColor,
```
Comment (MadCcc) [inline components/slider/style/index.tsx:223]: 这个也是，应该是蓝色圈的颜色

File: docs/react/migrate-less-variables.en-US.md [hunk 9]
```diff
@@ -389,9 +389,33 @@ export default App;
...
+| `@slider-margin` | - | Could be customized with `className` or `style` |
+| `@slider-rail-background-color` | `railBg` | - |
+| `@slider-rail-background-color-hover` | `railHoverBg` | - |
+| `@slider-track-background-color` | `trackBg` | - |
+| `@slider-track-background-color-hover` | `trackHoverBg` | - |
+| `@slider-handle-border-width` | `handleLineWidth` | - |
+| `@slider-handle-background-color` | - | Deprecated |
+| `@slider-handle-color` | `handleColor` | - |
+| `@slider-handle-color-hover` | `handleActiveColor` | - |
+| `@slider-handle-color-focus` | - | Deprecated |
```
Comment (MadCcc) [inline docs/react/migrate-less-variables.en-US.md:408]:
```suggestion
| `@slider-handle-color-focus` | `handleActiveColor` | - |
```

**Review by MadCcc — APPROVED** [review]

**Pull request merged**